index.js

```diff
@@ -4,7 +4,34 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let x, sum = 0;
```

Comment:
Could you pick a variable name that gives readers a better idea of the value stored in `x`?